umbral/pre.py

```diff
@@ -65,7 +66,7 @@ def __init__(self,
         self.point_v = point_v
         self.bn_sig = bn_sig
 
-        self._attached_cfrags = list()    # type: list
+        self._attached_cfrags = set()    # type: list
```

Comment:
```suggestion
        self._attached_cfrags = set()    # type: set
```
Just in case mypy complains later.